index.js

```diff
@@ -22,4 +22,38 @@ addTaskButton.addEventListener("click", () => {
   });
 
   console.log(tasks);
+
+  taskTable = document.getElementById("task-table");
+  const TaskDetails = document.createElement("tr");
+  TaskDetails.className = "table-box-row";
+
+  for (let taskArrayIndex in tasks) {
+    for (
+      let taskObjectIndex = 0;
+      taskObjectIndex < Object.values(tasks[taskArrayIndex]).length;
+      taskObjectIndex++
+    ) {
+
+      TaskDetails.innerHTML =
+        `<td class="table-box-color-tag-job">` +
+        "" +
+        `</td>` +
+        `<td class="table-box-task" id="show-task-title">` +
+        Object.values(tasks[taskArrayIndex])[1] +
+        `</td>` +
```

Comment:
concatenate the strings like this. take this an example.
```js 
`<div>
   <span>My name is ${name} and I am a ${job}</span>
</div>`
``` 
For more info: https://developer.chrome.com/blog/es6-template-strings/

Reply:
I love it.Thankssssss <3